src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -92,6 +92,9 @@ private static boolean isValidPath(Path filePath) {
      * @throws StorageOperationException if there were errors converting and/or storing data to file.
      */
     public void save(AddressBook addressBook) throws StorageOperationException {
+        if (!path.toFile().exists()) {
+            throw new StorageOperationException("Storage file not found.");
+        }
```

Comment:
This works for the exercise but do consider user experience when you're handling exceptions. Making a new exception class could be worth considering as well.